resources/public/api/conf/1.0/checkpoints_list.yaml

```
@@ -9,7 +9,7 @@ get:
     - Checkpoint for Vendor Data
 
   security:
-    - User-Restricted:
+    - Application-Restricted:
         - write:self-assessment
```

I think the list endpoint should be 
```suggestion
        - read:self-assessment
```
since it only retrieves (instead of modifying) any data. I can update the tech spec to reflect this